lib/path-handlers.js

```diff
@@ -47,6 +50,54 @@ function get_path(req, res, next) {
     });
 }
 
+
+function get_dir(req, res, next) {
+    var directories = [], juttles = [], relative_path, abs_path;
+
+
+    Promise.try(function() {
+        // massage the heck out of these paths
+        relative_path = req.query.path || '.';
+        relative_path = relative_path[0] === '/' ? relative_path.substring(1) : relative_path;
+        relative_path = path.normalize(relative_path);
```

Comment:
I could but I really wanted to limit the amount of code I had to write for this. I definitely need the /.. so people can travel back from a subdir. I guess the concern could be that someone could check for existence of a directory before the root a la `/../random-dir-exists`